tests/basics/BuiltinsTest.py

```diff
@@ -524,3 +524,9 @@ def __complex__(self):
     print("Oops, must not happen.")
 else:
     print("Oops, must not happen.")
+
+
+print("Tests for abs():")
+print(abs(-1000000**10))
+print(abs(len([1, 2, 3])))
```

Comment:
That `len()` will statically be optimized to `3` and make the test case relatively boring.

You would need to cover at least one exception case, static and one that Nuitka doesn't see through.

Reply:
I have added the exception case. I'm not sure about static and one that Nuitka doesn't see through cases.